src/Command.ts

```diff
@@ -27,6 +28,10 @@ const executors: _.Dictionary<(i: Invocation, a: string[]) => void> = {
     },
     clear: (invocation: Invocation, args: string[]): void => {
         setTimeout(() => invocation.terminal.clearInvocations(), 0);
+    },
+    exit: (invocation: Invocation, args: Array<string>): void => {
+        var application = Application.getInstance();
+        application.removeTerminal(application.activeTerminal);
```

Comment:
If there are multiple terminals, it doesn't re-render `ApplicationComponent` after removing the terminal.